test/ProgressOnderwijsUtils.Test/SortColumnTest.cs

```diff
@@ -13,21 +13,18 @@ public void CheckEquals()
             Assert.Equal(new ColumnSort("test", SortDirection.Asc), new ColumnSort("ziggy", SortDirection.Asc).WithDifferentName("test"));
             Assert.Equal(new ColumnSort("test", SortDirection.Asc), new ColumnSort("Test", SortDirection.Asc));
 
-            Assert.NotSame(new ColumnSort("abc", SortDirection.Asc), new ColumnSort("abc", SortDirection.Asc));
```

Comment:
Makes no sense, because ColumnSort is a value type.

Reply:
He?  Hoe... heeft dit ooit gewerkt?

Reply:
Ah zo
`same` betekent reference equality.

(Nou weet ik weer waarom ik die nunit assertion madness zo irritant vond)